php/5.6/node/8/scripts/docker-as-symfony

```diff
@@ -0,0 +1,440 @@
+#!/usr/bin/env bash
+set -e
+
+# Environement variables
+: ${DATABASE_URL:="mysql://root:root@db/drupal"}
+: ${DATABASE_DUMP:="/var/backups/db-reset.sql"}
+: ${SMTP_HOST:="mail:1025"}
+: ${REQUIRED_DIRECTORIES:=""}
+: ${LOCK_TIME:=""}
+: ${TEST_SERVER_PORT:="8888"}
+
+# Script variables
+DATABASE_HOST=$(a=${DATABASE_URL#*@}; echo ${a%/*})
+DATABASE_NAME=${DATABASE_URL##*/}
+DATABASE_USER=$(a=${DATABASE_URL#*//}; echo ${a%:*})
+DATABASE_PASSWORD=$(a=${DATABASE_URL#*//}; b=${a#*:}; echo ${b%@*})
+SYMFONY_IS_INSTALLED=0
+SCRIPT_NAME=$(basename $0)
+
+# Print help
+function print_help() {
+  echo "usage: $SCRIPT_NAME <action> [options]
+
+  * bootstrap
+
+    Install Symfony and dependencies. Require config export, do not bootstrap a new projet.
+
+    --skip-dependencies      # Do not run composer and yarn install
+    --skip-install           # Do not run Drupal install (only if arealdy installed)
+    --skip-db-reset          # Do not reset database using SQL dump (only when --skip-install)
+    --install-only           # Same as --skip-dependencies --skip-styleguide-build
+
+  * setup
+
+    Ensure Symfony is porper setup (parameters.yml, mail settings, ...).
+
+    --with-dependencies      # Run composer and yarn install
+
+  * db-reset
+
+    Reset database using SQL dump made after bootstrap.
+
+    --update-dump            # Update database dump (include updated Drupal config, but not
+                             # default content)
+
+  * db-dump
+
+    Create a database dump that can be used as with db-restore (not with db-reset).
+
+    --file=<path>            # Path to dump file (not required)
+
+  * db-restore
+
+    Restore database from dump created with db-dump.
+
+    --file=<path>            # Path to dump file (not required)
+
+  * apache-server
+
+    Run web server using 'apache'. Reset database if not present.
+
+    --with-db-reset          # Reset database before launch server
+    --cache                  # Do not invalidate opcache (server must be restarted
+                             # reload cache)
+    --help                   # Display apache help
+    -- <...>                 # any apache valid args
+  "
+}
+
+# Setup parameters.yml file, database connection and other setup stuff
+function setup() {
+  printf "\e[1;35m* Update parameters.yml file.\e[0m\n"
+
+  # Populate database config
+  sed -ri -e "s@database_host:.+@database_host: ${DATABASE_HOST}@g" /var/www/app/config/parameters.yml
```

Comment:
I don't think this is the way to go. Symfony are able to use env variables. Plus, `app/config/parameters.yml` is specific to Symfony 2.x and some 3.x versions.

Reply:
So it's a valid way to it in some Symfony versions. My goal is to detect later if it's a Symfony version that supports env, and ignore that in this case.